### PR TITLE
Use picture tag to fallback to png when the browser doesn't support WEBP

### PR DESCRIPTION
**Pull request by broskoski**
The idea here is to fallback to our original stupid PNGs when the browser doesn't support WEBP AND when the image is an `Image` (which is the only case where it matters). 

Once all browsers are equal, this code shall be removed.

### CODE REVIEW

**Comment by dzucconi** [comment]
The proxy should already handle this (content negotiation) and serve accordingly. It's not?

**Comment by broskoski** [comment]
It does, this is just about falling back to transparent PNGs on Image blocks in Safari (since JPEGs flatten transparency wrong).

File: src/extension/src/components/ExtensionLogin/index.jsx [hunk 1]
```diff
@@ -73,9 +73,9 @@ class Login extends Component {
               me.authentication_token
             )
           } catch {
-            // browser.storage.local.set({
-            //   authentication_token: me.authentication_token,
-            // })
+            browser.storage.local.set({
+              authentication_token: me.authentication_token,
+            })
```
Comment (dzucconi) [inline src/extension/src/components/ExtensionLogin/index.jsx:78]: What's this about?

File: package.json [hunk 2]
```diff
@@ -184,6 +184,7 @@
     "react-split-pane": "^0.1.89",
     "react-stripe-elements": "^2.0.1",
     "react-transition-group": "^1.2.0",
+    "react-use-webp-support-check": "^1.0.5",
```
Comment (dzucconi) [inline package.json:187]: This is added but then never imported

File: src/v2/components/Cell/components/Konnectable/components/KonnectableGeneric/index.tsx [hunk 3]
```diff
@@ -25,17 +25,24 @@ interface Props extends BoxProps {
...
   mode: Mode
+  fallbackToProxyJpeg?: boolean
 }
 
 export const KonnectableGeneric: React.FC<Props> = ({
   src,
   srcs,
   title = null,
   mode = Mode.RESTING,
+  fallbackToProxyJpeg,
```
Comment (dzucconi) [inline src/v2/components/Cell/components/Konnectable/components/KonnectableGeneric/index.tsx:36]: Why a prop?

File: src/v2/components/Cell/components/Konnectable/components/KonnectableImg/index.tsx [hunk 4]
```diff
@@ -12,7 +12,8 @@ const Container = styled(Box).attrs({
   width: 100%;
   height: 100%;
 
-  > img {
+  img,
+  picture {
```
Comment (dzucconi) [inline src/v2/components/Cell/components/Konnectable/components/KonnectableImg/index.tsx:16]: `> img, > picture`

**Comment by dzucconi** [comment]
Yeah I mean... this is a clever workaround, I just don't think it's warranted.

With the lack of `srcSet` it's objectively worse for those users: having flattened transparent pngs (which I still don't view as a 'bug') is obviously much worse than just having improperly sized images. This also presents problems with adopting the proxy elsewhere.

**Comment by broskoski** [comment]
CLOSING

**Pull request closed without merging**